contents/docs/data/identify.mdx

```diff
@@ -0,0 +1,466 @@
+```
+
+```php
+PostHog::alias(array(
+  'distinctId' => 'distinct_id',
+  'alias' => 'alias_id'
+));
+```
+
+```ruby
+posthog.alias({
+  distinct_id: "distinct_id",
+  alias: "alias_id",
+})
+```
+
+```go
+client.Enqueue(posthog.Alias{
+  DistinctId: "distinct_id",
+  Alias: "alias_id",
+})
+```
+
+```java
+posthog.alias("distinct_id", "alias_id");
+```
+
+```bash
+curl -v -L --header "Content-Type: application/json" -d '{
+    "api_key": "<ph_project_api_key>",
+    "properties": {
+        "distinct_id": "distinct_id_one",
+        "alias": "any_other_id"
+    },
+    "timestamp": "2020-08-16 09:03:11.913767",
+    "event": "$create_alias"
+}' https://app.posthog.com/capture/
+```
+
+</MultiLanguage>
+
+There are two requirements when assigning an `alias_id`:
+
+1. It cannot be associated with more than one `distinct_id`.
+2. The `alias_id` **must not** be the `distinct_id` of any other user. For example:
+
+<MultiLanguage selector="tabs">
+
+```node
+client.identify({
+    distinctId: 'distinct_id_one',
+    properties: {},
+// ❌ Not possible. 
+// You cannot use distinct_id_one as an alias for any_other_id
+client.alias({
+    distinctId: 'any_other_id',
+    alias: 'distinct_id_one',
+})
+```
+
+```javascript
+posthog.identify('distinct_id_one');
+// ❌ Not possible. 
+// You cannot use distinct_id_one as an alias for any_other_id
+client.alias({
+    distinctId: 'any_other_id',
+    alias: 'distinct_id_one',
+})
+```
+
+```python
+posthog.identify('distinct_id_one',{})
+# ❌ Not possible
+# You cannot use distinct_id_one as an alias for any_other_id
+posthog.alias('distinct_id_one', 'any_other_id');
+```
+
+```php
+PostHog::identify(array(
+  'distinctId' => 'distinct_id_one',
+  'properties' => array()
+));
+# ❌ Not possible
+# You cannot use distinct_id_one as an alias for any_other_id
+PostHog::alias(array(
+  'distinctId' => 'any_other_id',
+  'alias' => 'distinct_id_one'
+));
+```
+
+```ruby
+posthog.identify({
+  distinct_id: "distinct_id_one",
+  properties: {}
+})
+# ❌ Not possible
+# You cannot use distinct_id_one as an alias for any_other_id
+posthog.alias({
+  distinct_id: "any_other_id",
+  alias: "distinct_id_one",
+})
+```
+
+```go
+client.Enqueue(posthog.Identify{
+  DistinctId: "distinct_id_one",
+  Properties: posthog.NewProperties()
+})
+// ❌ Not possible
+// You cannot use distinct_id_one as an alias for any_other_id
+client.Enqueue(posthog.Alias{
+  DistinctId: "any_other_id",
+  Alias: "distinct_id_one",
+})
+```
+
+```java
+posthog.identify(
+  "distinct_id_one",
+  new HashMap<String, Object>()
+);
+// ❌ Not possible
+// You cannot use distinct_id_one as an alias for any_other_id
+posthog.alias("any_other_id", "distinct_id_one");
+```
+
+</MultiLanguage>
+
+## Using identify on the frontend
+
+In your frontend, all events within PostHog are associated with a specific person:
+1. Either an **anonymous user** – i.e., if `identify` has not been called.
+2. Or a **recognized user** – i.e., `identify` has been called..
```

Comment:
I feel slightly uncomfortable about introducing the term "recognized user", but maybe its okay since it's only purpose is to contrast it with an anonymous user